.github/workflows/stale.yml

```diff
@@ -1,8 +1,9 @@
 name: 'Close stale issues'
 on:
-  pull_request:
-    branches:
-      - master
+  # uncomment if you want to test changes to this file in PRs CI
```

Comment:
```suggestion
  # NOTE: uncomment if you want to test changes to this file in PRs CI
```

Reply:
Well 😄 https://github.com/Project-OSRM/osrm-backend/pull/6986#issuecomment-2218328050

Sorry for this...

Reply:
I'll create a separate one.